odoo/addons/test_lint/tests/test_i18n.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import re
+
+from . import lint_case
+
+from odoo import tools
+
+_logger = logging.getLogger(__name__)
+
+
+class TestI18n(lint_case.LintCase):
+    # Assumption: Text starting with an uppercase letter is likely supposed to be translated.
+    PROPS_RE = re.compile(
+    r"""
+        (
+            <[A-Z][^>]+\s+                  # Match a component opening tag with a capital letter, followed by attributes
+            (
+                [a-zA-Z-]+="[^"]*"\s+       # Match attribute name="value" patterns
+            )*                              # Zero or more such attributes
+        )
+        (?!t-)                              # Negative lookahead to exclude attributes starting with t-
+        (
+            [a-zA-Z-]+                      # Match attribute name
+            =                               # Match the equal sign
+            "                               # Match the opening quote
+            '                               # Match the inner single quote
+            [A-Z](\'|[^'])*?                # Match a capital letter followed by anything
+            [a-z]                           # Ensure there is at least one lowercase letter
```

Comment:
It would be interesting to explain why you added this constraint (avoid matching technical constants such as "ROW")